src/config/index.ts

```diff
@@ -339,7 +370,19 @@ if (acgConfig) {
     // Kafka settings
     data.kafka = {
         host: clowdAppConfig.kafka.brokers[0].hostname,
-        port: clowdAppConfig.kafka.brokers[0].port.toString()
+        port: clowdAppConfig.kafka.brokers[0].port.toString(),
+        sasl: {
+            username: clowdAppConfig.kafka.brokers[0].sasl.username,
+            password: clowdAppConfig.kafka.brokers[0].sasl.password
+        }
+    };
+
+    const tmpobj2 = tmp.fileSync({mode: 0o644, prefix: 'prefix-', postfix: '.txt'});
```

Comment:
I think app-common-js can do this for you. I'm not good with javascript, but take a look here and see what you think:
https://github.com/RedHatInsights/app-common-js/blob/f30f50195d436f4b3e6d429476c0de9bafd33c88/index.js#L12

Reply:
@SteveHNH Oh!  Yeah, so the reason I did that there is because the app-common-js lib doesn't work with the remediations-consumer code since it's written in Typescript 😅.

Reply:
ah! I can't read. Thanks for clarifying